components/push/storage/src/db.rs

```diff
@@ -171,6 +175,23 @@ impl Storage for PushDb {
         )?;
         Ok(affected_rows == 1)
     }
+
+    fn get_meta(&self, key: &str) -> Result<Option<String>> {
+        // Get the most recent UAID (which should be the same value across all records,
+        // but paranoia)
+        self.try_query_one(
+            "SELECT value FROM meta_data where key = :key limit 1",
+            &[(":key", &key)],
+            true,
+        )
+        .map_err(|e| ErrorKind::StorageSqlError(e).into())
+    }
+
+    fn set_meta(&self, key: &str, value: &str) -> Result<()> {
+        let query = "INSERT or REPLACE into meta_data (key, value) values (:k, :v)";
+        self.execute_named(query, &[(":k", &key), (":v", &value)])?;
```

Comment:
Nit `execute_named_cached` (missed this last time, sorry).

Reply:
crap, sorry, you're right.